README.md

```diff
@@ -1,16 +1,54 @@
 # Release Integration Tests
 
 
-## 1. FBC Test (happy path):
-The purpose of this release test (FBC) is to verify the functionality and stability of feature in RHTAP.
-Test is a "happy-path" test and it's running on the internal RHTAP stage cluster.
+## 1- Fbc happy path e2e-test ##
+
+ This section provides instructions on running the FBC e2e test with the provided script
+
+   ## Test Steps and Expected Result
+
+   1. **Pre-Running Test:** The following setup actions need to be completed before running the test script.
+      
+      The test is running against RHTAP-Stage, so a user who wants to monitor the test running through the UI or the CLI. 
+      will need access to the cluster and the workspaces *release-dev-team* and *release-managed-team*. 
+      
+      Our test expect to run on both workspaces above, and it's expected (based on the flow agreed), namespaces dev and managed exist, policy *(fbc-policy)*, service account, secrets already deployed on the two dedicated workspaces 
```

Comment:
`s/Our test expect/Our test expects/`

Reply:
Can you use the full namespace names instead of the shortened versions here?

Reply:
I think this would be better like

The test expects to run in the workspaces mentioned above. It is expected that the dev-release-team-tenant and managed-release-team-tenant namespaces exist with the EnterpriseContractPolicy, ServiceAccount and secrets already deployed in them.